core/modules/admin_bar/admin_bar.inc

```diff
@@ -752,6 +752,10 @@ function admin_bar_links_locale() {
   $links['locale'] = array(
     '#title' => isset($language->native) ? $language->native : $language->name,
     '#attributes' => array('class' => array('admin-bar-locale-links')),
+    '#options' => array(
+      'html' => TRUE,
```

Comment:
I don't think `'html'` is needed here, and might actually cause an XSS issue if the language name (which is configurable) is printed without sanitization.

Reply:
Oops, right, that unnecessary and possibly dangerous. Removed. :+1: